src/layouts/InterfaceLayout/InterfaceLayout.vue

```diff
@@ -354,6 +364,20 @@ export default {
             })
             .catch(KeepkeyWallet.errorHandler);
           break;
+        case BC_VAULT:
+          BCVaultWalletInstance.init()
```

Comment:
why is this defined in both hardwaremodal and here?

Reply:
for the switch address